tests/unit/command/test_imp_url.py

```diff
@@ -44,3 +49,57 @@ def test_import_url_no_exec(mocker):
     assert cmd.run() == 0
 
     m.assert_called_once_with("src", out="out", fname="file", no_exec=True)
+
+
+def test_import_url_https(dvc):
+    cli_args = parse_args(
+        ["import-url", "https://data.dvc.org/get-started/data.xml"]
+    )
+    assert cli_args.func == CmdImportUrl
+
+    cmd = cli_args.func(cli_args)
+    assert cmd.run() == 0
+
+
+# The first file id is a publicly available file.
+# The second, instead, requires authentication, which depend on a proper
+# gdrive-user-credentials.json. The absence of the file skips the test.
+@pytest.mark.parametrize(
+    "path, auth",
+    [
+        ("1nKf4XcsNCN3oLujqlFTJoK5Fvx9iKCZb", False),
+        ("1syA-26p7tehWyUiMPPk_s0hsFN0Nr_kX", True),
+        ("16onq6BZiiUFj083XloYVk7LDDpklDr7h/dir/data.txt", True),
+        ("16onq6BZiiUFj083XloYVk7LDDpklDr7h/dir", True),
+    ],
+)
+def test_import_url_gdrive(dvc, path, auth):
```

Comment:
This doesn't seem like a unit test. Is there a reason why we can't just test it in `tests/func/test_import_url.py::test_import_url_dir` like we do with all other cloud types?

Reply:
If I understood correctly, there is no mock service for gdrive. So, I wrote the test relying on the actual service.

Reply:
There is no mock, correct. There is a `gdrive` fixture though, with real gdrive (e.g. gs also relies on real google-cloud-storage). E.g. see tests/remotes/http.py , where it implements the minimal required methods (mkdir, write_text, write_bytes) for the fixture to be able to use it in `test_import_url`. `gdrive` fixture will handle skipping for you too automatically.

Reply:
I got it. I'll adapt this test to follow the format of those in `tests/func/test_import_url.py::test_import_url_dir`.